### ISSUE TITLE
[Bug Report] el-image-viewer causes SSR to fail because of missing window existence check

### ISSUE THREAD

**Issue by fmakdemir** [ElemeFE/element#16737]
### Element UI version
2.11.0

### OS/Browsers version
All

### Vue version
2.6.10

### Reproduction Link
https://github.com/fmakdemir/element-ui-image-viewer-ssr

### Steps to reproduce
* `yarn install`
* `yarn dev`
* Visit the given URL (http://localhost:3000/ if port is available)

### What is Expected?
Greeted with default page for nuxt and element ui setup

### What is actually happening?
Error page saying `window is not defined` pointing to `element-ui.common.js` file. The part that gives error is inside (new?) `element-ui/packages/image/src/image-viewer.vue` component line 69.

Needs just a simple fix of adding typeof `window !== 'undefined'` check to the statement:
```js
const isFirefox = typeof window !== 'undefined' && !!window.navigator.userAgent.match(/firefox/i);
```

**Comment by yoyo837** [comment]
Same here... again. #15261 #15315

**Comment by fmakdemir** [comment]
This is not the same issue actually, they are both caused by SSR but this one simply needs a check for if window is present (where in many other places it is already done).

**Comment by shlima** [comment]
+1

**Comment by luckyCao** [comment]
#16747

**Comment by iamkun** [comment]
fixed in 2.11.1
